### PR TITLE
rage.ftl ru translation

### PR DESCRIPTION
**Pull request by hexanova**
line count is not 1:1

i think this could use another formatting pass but i think it's good otherwise.. lmk 👍

### CODE REVIEW

**Comment by str4d** [comment]
> line count is not 1:1

That doesn't matter; what is more important is that the line lengths are approximately the same. I'll give it a pass if necessary.

**Comment by hexanova** [comment]
> > line count is not 1:1
> 
> That doesn't matter; what is more important is that the line lengths are approximately the same (except for the manpage strings, where paragraphs are unwrapped). I'll give it a pass if necessary.

gotcha. i think the manpage was where i was the most uncertain. the lines before that section are pretty close.

File: rage/i18n/ru/rage.ftl [hunk 1]
```diff
@@ -56,454 +56,460 @@
...
+err-failed-to-write-output = Не удалось записать в выходной файл: {$err}
+err-identity-ambiguous = {-flag-identity} требует либо {-flag-encrypt}, либо {-flag-decrypt}.
+err-mixed-encrypt-decrypt = {-flag-encrypt} не может использоваться вместе с {-flag-decrypt}.
+err-passphrase-timed-out = Истекло время ожидания ввода пароля.
+err-same-input-and-output = Входной и выходной файлы совпадают '{$filename}'.
+
+err-ux-A = {-rage} не сделал то, что вы ожидали? Могла ли быть полезнее ошибка?
+err-ux-B = Сообщите нам
+# Поставьте здесь пробелы (len(A) - len(B) - 32).
 err-ux-C = {"                          "}
```
Comment (str4d) [inline rage/i18n/ru/rage.ftl]: For reference, you don't need to translate the comment (though it is fine to do so for the benefit of future `ru` translators); it is there to tell you as the translator what to do with this string. In this case, `err-ux-A` has length 65 (after substituting `{-rage}`), and `err-ux-B` has length 12, so `err-ux-C` should contain 21 space characters.

**Review by str4d — APPROVED** [review]
Looks great, thank you! I've pushed a commit that adjusts some line wrapping, removes trailing spaces, and fixes a couple of bugs (some `{reference-ids}` were translated, breaking them and causing Fluent to fall back to the English strings).

**Comment by hexanova** [comment]
> Looks great, thank you! I've pushed a commit that adjusts some line wrapping, removes trailing spaces, and fixes a couple of bugs (some `{reference-ids}` were translated, breaking them and causing Fluent to fall back to the English strings).

thank you, happy to help! glad it wasn't too bad, definitely felt that rush to make the release 😅

will remember this if more work is needed in the future.

**Pull request merged**